TestTools/StreamChatTestMockServer/MockServer/ChannelResponses.swift

```diff
@@ -67,6 +67,10 @@ public extension StreamMockServer {
         server.register(MockEndpoint.truncate) { [weak self] request in
             self?.channelTruncation(request)
         }
+        server.register(MockEndpoint.sync) { request in
+            // TODO:
```

Comment:
Maybe we can write here what needs to be done?

Reply:
Done 👍